Toolset/libraries/revideextensionlibrary.livecodescript

```diff
@@ -838,35 +862,82 @@ function revIDEExtensionsOrderByDependency pExtensions
    return __dependencyOrder(tDependencies, pExtensions)
 end revIDEExtensionsOrderByDependency
 
+command revIDEExtensionFetchSourceFromFolder pFolder, @rSource, @rType
+   local tFiles
+   put files(pFolder) into tFiles
+   
+   local tExtFile, tType
+   filter tFiles with "*.lcb" into tExtFile
+   put "lcb" into tType
+   if tExtFile is empty then
+      put "lcs" into tType
+      filter tFiles with regex pattern ".*\.livecode(script)?$" into tExtFile
+   end if
+
+   put tExtFile into rSource
+   put tType into rType
```

Comment:
@livecodeali if `tExtFile` is still empty here this would set `rSource` & `rType` incorrectly. Perhaps a type of `none`?